plugins/mcreator-core/procedures/entity_makeride.json

```diff
@@ -0,0 +1,39 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "sourceentity",
+      "check": "Entity"
+    },
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    }
+  ],
+  "inputsInline": true,
+  "previousStatement": null,
+  "nextStatement": null,
+  "colour": "%{BKY_LOGIC_HUE}",
```

Comment:
Change this color to the color of entity blocks. LOGIC_HUE is reserved for output blocks that return logic values.

Reply:
Done!